wiki/Help_centre/Store/en.md

```diff
@@ -2,124 +2,48 @@
 tags:
   - store
   - product
-  - keyboard
-  - tablet
-  - area
-  - device
-  - setup
-  - pendrive
+  - keycap
+  - keycaps
+  - deskmat
+  - deskmats
 ---
 
 # osu!store and merchandise
 
-*Main page: [Help centre](/wiki/Help_centre)*\
-*Note: These products are no longer available for purchase from the osu!store.*
+*Main page: [Help centre](/wiki/Help_centre)*
 
-Having trouble with a product from the osu!store? Check if there's an available solution to your problem!
+This article only applies to products purchased from the [osu!store](https://osu.ppy.sh/store/listing).
 
-## osu!keyboard {id=osu!keyboard}
+## I need to change my address!
```

Comment:
i recommend adding new custom identifiers, such that section links always work even for people with different website language settings